packages/nvidia_nat_vanna/src/nat/plugins/vanna/vanna_utils.py

```
@@ -0,0 +1,852 @@
+            records = await self.async_milvus_client.query(
+                collection_name=collection_name,
+                output_fields=[output_field],
+                limit=self.milvus_search_limit,
+            )
+            for record in records:
+                record_list.append(record[output_field])
+        except Exception as e:
+            logger.error(f"Error retrieving {collection_name}: {e}")
+        return record_list
+
+    async def get_similar_question_sql(self, question: str, **kwargs) -> list:
```

Ruff (0.14.4)

Unused method argument: `kwargs`

(ARG002)